src/ajax.js

```diff
@@ -615,7 +611,8 @@ jQuery.extend( {
 			// Add or update anti-cache param if needed
 			if ( s.cache === false ) {
 				cacheURL = cacheURL.replace( rantiCache, "$1" );
-				uncached = ( rquery.test( cacheURL ) ? "&" : "?" ) + "_=" + ( nonce++ ) + uncached;
+				uncached = ( rquery.test( cacheURL ) ? "&" : "?" ) + "_=" +
+					( nonce.nonce++ ) + uncached;
```

Comment:
Assuming the `nonce` property name doesn't matter, you might get a little more compression by renaming it (e.g., `nonce.value++` or `nonce.data++` or `nonce.length++` or `nonce.expando++` or `nonce.guid++`).

Reply:
Thanks, I'll try.

Reply:
`guid` reduces size differences from `-10` bytes to `-12` bytes, I'll use that.